model/metricset.go

```diff
@@ -68,6 +68,11 @@ type Metricset struct {
 
 	// Samples holds the metrics in the set.
 	Samples []Sample
+
+	// TimeseriesInstance holds an optional identifier for the timeseries
+	// instance, such as a hash of the labels used for aggregating the
+	// metrics.
+	TimeseriesInstance string
```

Comment:
shouldn't this be called `TimeseriesInstanceID` or similar?

Reply:
My intention was to keep it in line with the field name produced, but I agree that it would be clearer with ID in the name. I'll change to the name you suggest.